lib/elixir/lib/agent.ex

```diff
@@ -241,7 +241,7 @@ defmodule Agent do
   and the start function will return `{:error, :timeout}`.
 
   If the `:debug` option is present, the corresponding function in the
-  [`:sys` module](http://www.erlang.org/doc/man/sys.html) will be invoked.
+  [`:sys` module](`:sys`) will be invoked.
```

Comment:
+1 for this and similar changes.

I think it was useful to see the erlang.org URL before, but now that `iex> h :sys` works, there's imho less of a need for that.

In terms of rendering `` [`:sys` module](`:sys`) `` link in IEx, I think we should show it as just `` `:sys` module ``, stripping the "url" part. This is something we should do for another reason and that is OTP docs chunks contain their own internal links like `` stdlib:sys#gen_server ``:

<img width="677" alt="image" src="https://user-images.githubusercontent.com/76071/108307496-255c8280-71ae-11eb-8e8a-4454b4508e0a.png">

we should either transform the links to something more clear or strip them, I think it's totally reasonable to just strip them.

Even links to named anchors like `` [whatever](#foo) `` are arguably not useful to show, so I'd strip the URL part too.

Putting this together, I'd consider only displaying the URL if it's a http/https URL.

WDYT?